src/yunohost/user.py

```diff
@@ -118,33 +115,27 @@ def user_create(auth, username, firstname, lastname, mail, password,
     })
 
     # Validate uniqueness of username in system users
-    try:
-        pwd.getpwnam(username)
-    except KeyError:
-        pass
-    else:
+    all_existing_usernames = {x.pw_name for x in pwd.getpwall()}
```

Comment:
Didn't know this `{ }` list-comprehension-like thing.. Is this for sets/unordered list ?